appveyor.yml

```diff
@@ -0,0 +1,35 @@
+build: false
+skip_branch_with_pr: true
+# Actual shallow *clone*. The shallow_clone appveyor option downloads
```

Comment:
This comment should be removed; a shallow clone is not being performed in this build and the comment is misleading. Otherwise, a shallow clone can performed here with `clone_depth: <number>`.